packages/analytics-js/src/components/utilities/json.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Utility method for excluding null values in JSON
+ * @param {*} key key in the object
+ * @param {*} value value in the object
+ * @returns
+ */
+const replaceNullValues = (key: string, value: any) => value ?? undefined;
```

Comment:
This will also discard the empty string. Right now empty string is allowed. So this will be a breaking change

Reply:
It's a null coalescing operator. It only disallows `null` and `undefined` values.